articles/android-mdm-setup.md

```diff
@@ -20,7 +20,6 @@ When you select **Connect Android Enterprise**, Fleet will open the Google signu
 4. Head to your [Google Admin console](https://admin.google.com).
 5. From the side menu, select [Devices > Mobile & endpoints > Settings > Third-party integrations](https://admin.google.com/ac/devices/settings/thirdparty).
 6. Select **Android EMM**, check **Enable third-party Android mobile management**, and then select **Manage EMM providers**.
```

Comment:
```suggestion
5. From the side menu, select [Devices > Mobile & endpoints > Settings > Third-party integrations](https://admin.google.com/ac/devices/settings/thirdparty).
6. Select **Android EMM**, check **Enable third-party Android mobile management**, and then select **Manage EMM providers**.
```

We're cutting step 7 for the same reason we're cutting "Add users from Microsoft to Google Workspace right?